File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.js
function solution(price) {
    let answer = price;
    
    if(price >= 500000) {
        answer = parseInt(price * 0.8);
    } else if(price >= 300000) {
        answer = parseInt(price * 0.9);
    } else if(price >= 100000) {
        answer = parseInt(price * 0.95);
    } 
       
    return answer;
}

File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.go
func solution(price int) int {
    var answer int = 0
        
    if price < 100000 {
        answer = price
    } else if 100000 <= price && price < 300000 {
        answer = int(float64(price) * 0.95)
    } else if 300000 <= price && price < 500000 {
        answer = int(float64(price) * 0.9)
    } else {
        answer = int(float64(price) * 0.8)
    }

    return answer;
}